templates/includes/sidebar.html

```diff
@@ -1,4 +1,10 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+  <li><a href="{% url 'taxi:driver-detail' pk=user.id %}">{{ user.username }}</a></li><br>
```

Comment:
Ensure that the URL pattern `taxi:driver-detail` is correctly defined in your URL configurations. This is necessary for the link to the driver's detail page to work properly.